Xamarin.Forms.Platform.Android/Renderers/EditorRenderer.cs

```diff
@@ -177,5 +182,18 @@ void OnKeyboardBackPressed(object sender, EventArgs eventArgs)
 			ElementController?.SendCompleted();
 			Control?.ClearFocus();
 		}
+
+        void UpdateMaxLength()
+        {
+            var currentFilters = Control?.GetFilters()?.ToList() ?? new List<IInputFilter>();
```

Comment:
Sort of an unspoken rule, that they tend not to use Linq, due to performance. Instead of Linq, use for loops, and copies. Also use a For loop instead of a ForEach loop, unless you have to use the item from the array more than once. That's an old mono perf issue, from my limited understanding.

Reply:
Noted and removed!